.travis/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+for i in `seq 1 4`; do
```

Comment:
What's the logic behind generating 4 CAs?

Reply:
During my testing, I noticed that re-running the same command uncovers issues with how the code does RPM rotations and RPM DB look-ups. So this exercises all this code for CA, server and client.